test/modules/generic-mqtt-tester/README.md

```diff
@@ -1,4 +1,22 @@
 # Generic MQTT Test Module
 
+## Description
 See description here:
-[link](src/tester.rs#L96)
\ No newline at end of file
+[link](src/tester.rs#L96)
+
+## Env Vars
+
+Defaults can be seen here:
+[link](config/default.json)
+
+|Env Var| Description|
+|----------|------------|
+|TEST_SCENARIO | Specifies the mode that the test module runs in|
+|TRC_URL| Specifies the url of the TestResultCoordinator (TRC). Needs manual specification if deploying on a node which does not run the TRC. Othewise will defaul accordingly.|
+|TRACKING_ID| Specifies the tracking id for which to tag reports to the TRC. Needed so the TRC can verify test results are truly coming from the same test.|
+|TEST_START_DELAY| Specifies the delay before the test logic starts running. Needed to give time for the environment to spin up and get everything ready.|
```

Comment:
In the future, is there anyway we can use signaling or a callback so that the test knows when it is safe to start, rather than relying on this delay period? If our tests never fail because of starting too early, then this is low priority, but if it occurs occasionally, then it's definitely worth considering.

Reply:
We considered this when architecting the TRC but we thought delay was fine. Over the time since it has been out we just have a large delay period and everything works fine.